src/web/app/pages-student/student-course-details-page/student-course-details-page.component.spec.ts

```diff
@@ -36,48 +37,55 @@ describe('StudentCourseDetailsPageComponent', () => {
   });
 
   it('should snap with populated fields', () => {
-    const courseAttributes: any = {
-      createdAt: '2019-03-12T09:13:29.272Z',
-      id: 'CS3281',
-      name: 'Thematic Systems',
-      timeZone: 'Asia/Singapore',
+    const student: Student = {
+      courseId: '1.1.c-demo2',
+      email: '1@1.com',
+      lastName: '1',
+      name: '1',
+      comments: '',
+      requestId: '5cb7d3320000b3b0cb4e5208',
```

Comment:
You don't need to have construct requestId?

Reply:
Yup removed.